Modules/Campaign/Database/Migrations/2023_02_16_143538_create_campaigns_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('campaigns', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->string('campaign_key')->unique();
+            $table->text('title');
+            $table->text('subject');
+            $table->text('preview_text');
+            $table->longText('email_design');
+            $table->string('scheduled_date')->nullable();
+            $table->enum('status',['draft', 'scheduled', 'completed'])->default('draft');
+            $table->foreign('user_id')->references('id')->on('users')->onDelete('CASCADE');
```

Comment:
Foreign key declaration should come after timestamps and be separated with line.